frontend/packages/kubevirt-plugin/src/components/vm-status/vm-status.tsx

```diff
@@ -0,0 +1,248 @@
+import * as React from 'react';
+import { PodKind, K8sResourceKind } from '@console/internal/module/k8s';
+import { OffIcon, UnknownIcon } from '@patternfly/react-icons';
+import {
+  PopoverStatus,
+  StatusIconAndText,
+  getNamespace,
+  getName,
+  ErrorStatus,
+  ProgressStatus,
+  PendingStatus,
+  SuccessStatus,
+} from '@console/shared';
+import { Progress, ProgressVariant, ProgressSize } from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+import { resourcePath } from '@console/internal/components/utils';
+import { POD_DETAIL_OVERVIEW_HREF } from '@console/internal/components/utils/href';
+import { PodModel } from '@console/internal/models';
+import { VirtualMachineModel } from '../../models';
+import { VM_DETAIL_EVENTS_HREF, CDI_KUBEVIRT_IO, STORAGE_IMPORT_PVC_NAME } from '../../constants';
+import { getLabels } from '../../selectors/selectors';
+import { getVMStatus } from '../../statuses/vm/vm';
+import {
+  VM_STATUS_V2V_CONVERSION_PENDING,
+  VM_STATUS_VMI_WAITING,
+  VM_STATUS_POD_ERROR,
+  VM_STATUS_IMPORT_ERROR,
+  VM_STATUS_V2V_CONVERSION_ERROR,
+  VM_STATUS_IMPORTING,
+  VM_STATUS_V2V_CONVERSION_IN_PROGRESS,
+  VM_STATUS_STARTING,
+  VM_STATUS_MIGRATING,
+  VM_STATUS_RUNNING,
+  VM_STATUS_OFF,
+  VM_STATUS_ERROR,
+} from '../../statuses/vm/constants';
+import { VMKind } from '../../types';
+
+import './vm-status.scss';
+
+const VIEW_POD_OVERVIEW = 'View pod overview';
+const VIEW_VM_EVENTS = 'View events';
+const IMPORTING_VMWARE_MESSAGE =
+  'The virtual machine is being imported from VMware. Disks will be converted to the libvirt format.';
+const IMPORTING_ERROR_VMWARE_MESSAGE = 'The virtual machine could not be imported from VMware.';
+const IMPORTING_MESSAGE =
+  'The virtual machine is being imported. Disks are being copied from the source image.';
+const IMPORTING_ERROR_MESSAGE = 'The virtual machine could not be imported.';
+const VMI_WAITING_MESSAGE = 'The virtual machine is waiting for resources to become available.';
+const STARTING_MESSAGE =
+  'This virtual machine will start shortly. Preparing storage, networking, and compute resources.';
+
+const getAdditionalImportText = (pod: PodKind): string => {
+  const labels = getLabels(pod, {});
+  const labelValue = labels[`${CDI_KUBEVIRT_IO}/${STORAGE_IMPORT_PVC_NAME}`];
+  return labelValue ? ` (${labelValue})` : null;
```

Comment:
```suggestion
  return labelValue ? `(${labelValue})` : null;
```

The extra space can be handled in JSX as `{' '}` if necessary.

String concatenation shouldn't be a concern of `getAdditionalImportText` function, but rather a concern of code that uses it.